docker-compose.yml

```diff
@@ -18,6 +18,9 @@ x-airflow-env:
     - AIRFLOW__API__AUTH_BACKEND=airflow.api.auth.backend.default
     - AIRFLOW__API__ENABLE_EXPERIMENTAL_API=True
     - AIRFLOW__SCHEDULER__MIN_FILE_PROCESS_INTERVAL=5
+    - AIRFLOW__WEBSERVER__SECRET_KEY='WmZHRmJwd1dCUEp6Xl4zVA=='
+    - AIRFLOW__WEBSERVER__SECRET_KEY_CMD='WmZHRmJwd1dCUEp6Xl4zVA=='
```

Comment:
the `_CMD` environment variables usually replace the non `_CMD` environment counter part by providing a command to get the value instead. That is to avoid secrets being visible in environment variables. (Not an issue for development)

Reply:
I am planing to define the secrets in flux cluster config.

Reply:
I could imagine that the Airflow Helm chart is already taking care of that?